test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Connection/ConnectionServiceTests.cs

```diff
@@ -83,7 +83,10 @@ public void DatabaseChangesAffectAllConnections()
             // All open DbConnections (Query and Default) should have initialDatabaseName as their database
             foreach (DbConnection connection in connectionInfo.AllConnections)
             {
-                Assert.Equal(connection.Database, initialDatabaseName);
+                if (connection != null && connection.State == ConnectionState.Open)
```

Comment:
why the database is closed here? can you please check? I know I  made  achange to the connection service so the urr start with "connection" and "dashboard" will be closed. I want to make sure it's not related

Reply:
Here, connectionInfo.AllConnections has all the connections it has made. So, it actually has 2 connections, one when created by `InitLiveConnectionInfo()` and one created when we execute the query. Only the connection that was used to execute the query remains open while the other one's closed. Also, that connection doesn't have a database name since we don't give any parameter to `InitLiveConnectionInfo()`, which is why it was failing.